crypto/keyring/keyring.go

```diff
@@ -64,13 +64,16 @@ type Keyring interface {
 	Delete(uid string) error
 	DeleteByAddress(address sdk.Address) error
 
-	// NewMnemonic generates a new mnemonic, derives a hierarchical deterministic
-	// key from that, and persists it to the storage. Returns the generated mnemonic and the key
-	// Info. It returns an error if it fails to generate a key for the given algo type, or if
+	// NewMnemonic generates a new mnemonic, derives a hierarchical deterministic key from that, and
+	// persists it to the storage. Returns the generated mnemonic and the key Info secured with the given passphrase.
```

Comment:
```suggestion
	// persists it to the storage. Returns the generated mnemonic and the key Info secured with the given passphrase.
```

No, this sounds wrong. The key is not secured with the given passphrase. The passphrase is used only in the Derive() function.